src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,19 @@
-export const UserInfo = () => {};
+import React from 'react';
+import { User } from '../../types/User';
+
+type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  const {
+    name,
+    email,
+  } = user;
```

Comment:
2 attributes can be in one line
```suggestion
  const { name, email } = user;
```